pkg/integration/docker_test.go

```diff
@@ -0,0 +1,86 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_dockerHubClient_GetImages(t *testing.T) {
+	hubClient := NewDockerHubClient("", "")
+	hubClient.client = &clientMockImages{}
+
+	organization := "dhis2"
+	images, err := hubClient.GetImages(organization)
+	require.NoError(t, err)
+
+	assert.True(t, slices.Contains(images, "core"))
```

Comment:
Testify has a huge list of assertions we can use. So IMHO we should not add another dependency unless its absolutely needed. Testify is already a pretty big library. As a purist 😋 I would only use go test and https://pkg.go.dev/github.com/google/go-cmp/cmp

Since we are using testify I would reach for 
https://pkg.go.dev/github.com/stretchr/testify/assert#ElementsMatch
since I assume you only expect a slice with 2 elements in it and you don't care about order. If you would only want to assert that a couple of elements are in there but not on the entire slice then https://pkg.go.dev/github.com/stretchr/testify/assert#Contains